examples/producer.java

```diff
@@ -0,0 +1,48 @@
+import java.util.concurrent.ExecutionException;
+import com.memphis.Headers;
+import com.memphis.Memphis;
+import com.memphis.MemphisConnectError;
+import com.memphis.MemphisError;
+import com.memphis.MemphisHeaderError;
+import com.memphis.MemphisSchemaError;
+import com.memphis.producer.Producer;
+
+public class Main {
+
+    public static void main(String[] args) throws ExecutionException, InterruptedException {
+        try {
+            Memphis memphis = new Memphis();
+            memphis.connect("<memphis-host>", "<application type username>", "<broker-token>").get();
+
+            Producer producer = memphis.createProducer("<station-name>", "<producer-name>");
```

Comment:
I believe this method returns also a completeAble future so you will need to call get() here as well